SprintCleaning/Assets/Scripts/Garbage.cs

```diff
@@ -9,6 +9,8 @@ public class Garbage : MonoBehaviour
 
     [Tooltip("The sound this piece of garbage plays on collect")]
     public AudioClip impact;
+
+    public bool _obstacle;
```

Comment:
Try to avoid public fields, since that's best practice. If it needs to be edited in the inspector, use `[SerializeField] private bool _obstacle`. If it needs to be read by another script but not changed, use `public bool Obstacle {get; private set; }`. If both, use `[field: SerializeField] public bool Obstacle {get; private set; }`.